File: c/006.c
#include <stdio.h>

int main() {
	int sum = 0;
	int sum_of_squares = 0;
	for (int i = 1; i <= 100; i++) {
		sum += i;
		sum_of_squares += i * i;
	}
	int square_of_sum = sum * sum;
	int difference = square_of_sum - sum_of_squares;
	printf("%d\n", difference);
}

File: go/006.go
package main

import "fmt"

func main() {
	sum := 0
	sumOfSquares := 0

	for i := 1; i <= 100; i++ {
		sum += i
		sumOfSquares += i * i
	}

	squareOfSum := sum * sum
	fmt.Println(squareOfSum - sumOfSquares)
}
